tests/conftest.py

```diff
@@ -108,7 +111,9 @@ def to_primitive(self, value, context=None):
 
             def to_native(self, value, context=None):
                 try:
-                    return datetime.datetime.fromtimestamp(float(value) / 1000000)
+                    return datetime.datetime.fromtimestamp(
+                        float(value) / 1000000, tz=tz.tzutc()
```

Comment:
Theses changes needed to happen because when I turned on validation for schematics the tests failed with:


```
____________________________________________________________________________________________________________ test_native_types _____________________________________________________________________________________________________________

TestModel = <class 'tests.conftest.TestModel'>, TestModel_table = None, dynamo_local = <dynamorm.local.DynamoLocal object at 0x10d374b90>

    def test_native_types(TestModel, TestModel_table, dynamo_local):
        DT = datetime.datetime(2017, 7, 28, 16, 18, 15, 48, tzinfo=dateutil.tz.tzutc())
    
        TestModel.put(
            {
                "foo": "first",
                "bar": "one",
                "baz": "lol",
                "count": 123,
                "when": DT,
                "created": DT,
            }
        )
>       model = TestModel.get(foo="first", bar="one")

tests/test_table.py:520: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
dynamorm/model.py:339: in get
    return cls.new_from_raw(item)
dynamorm/model.py:324: in new_from_raw
    return cls(partial=partial, **raw)
dynamorm/model.py:224: in __init__
    raw, partial=partial, native=True
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'tests.conftest.TestModelSchema'>, obj = {'bar': 'one', 'baz': 'lol', 'count': Decimal('123'), 'created': Decimal('1501229895000000'), ...}, partial = False, native = True

    @classmethod
    def dynamorm_validate(cls, obj, partial=False, native=False):
        try:
            inst = cls(obj, strict=False, partial=partial, validate=True)
        except (SchematicsValidationError, ModelConversionError) as e:
>           raise ValidationError(obj, cls.__name__, e.messages)
E           ValidationError: Validation failed for schema TestModelSchema. Errors: {'created': ValidationError([ErrorMessage(u'Time zone information required but not found.', None)])}

dynamorm/types/_schematics.py:32: ValidationError
```